app/src/main/res/values/strings.xml

```diff
@@ -6,4 +6,6 @@
     <string name="health_facility">Health Facility</string>
     <string name="monitor_js_multiple_event_legend">"(*)More than one survey exist in the same month"</string>
     <string name="dialog_pie_chart_label_explanation">"The outer circle shows you the overall completion.\n\nThe inner circle shows you the completion of mandatory questions, normally marked with an *. Only when 100% of the mandatory questions are completed, you can 'Mark Complete'"</string>
+    <string name="error_unauthorized">"Wrong username or password."</string>
+    <string name="title_error_unexpected">"Unexpected error"</string>
```

Comment:
These strings are in the wrong place. 

At the moment in malariapp, the strings are repeated in all the variants and its language resources to avoid change a term in all the variants.

Only the eds variant is ignored.

They should be in /src/hnqis/res/values-xx/string.xml for each language.

Other problem is that strings are not translated in all the languages on hnqis variant.

And the strings without translations are been stored in donottranslate.xml

I think the /src/main/res/values/strings.xml resources should be deleted, we can talk about it in the next meeting.